BigBaseV2/src/views/vehicle/view_lsc.cpp

```diff
@@ -725,9 +733,7 @@ namespace big
 							selected_color = color;
 							dashboard_color = color;
 
-							g_fiber_pool->queue_job([] {
-								VEHICLE::SET_VEHICLE_DASHBOARD_COLOR_(player_vehicle, dashboard_color);
-							});
+							VEHICLE::SET_VEHICLE_DASHBOARD_COLOR_(player_vehicle, dashboard_color);
```

Comment:
Calling natives outside of script threads is detected.

Use the fiber pool please.